nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/HotDeployer.java

```diff
@@ -43,16 +43,16 @@
     public HotDeployer(FeaturesRunner runner, RuntimeHarness harness) {
         this.runner = runner;
         this.harness = harness;
-        this.head = new DefaultDeployHandler();
+        head = new DefaultDeployHandler();
```

Comment:
Keep it for this commit, but in general such refactoring should be in separate commits.